examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb

```diff
@@ -154,7 +154,10 @@
     "# define the number of repeats for the hybrid solver to search for the optimal solution.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    # define the number of shots per task for the hybrid solver to search for the optimal solution.
</pre><p> </p><p>change to</p><p> </p><pre class="ql-syntax" spellcheck="false"># define the number of shots (or reads) per task on the D-Wave QPU
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>